packages/datagateway-common/package.json

```diff
@@ -102,6 +102,9 @@
     ]
   },
   "jest": {
+    "moduleNameMapper": {
+      "axios": "axios/dist/node/axios.cjs"
+    },
```

Comment:
Can we use the

```
"transformIgnorePatterns": [
      "node_modules/(?!axios)"
],
```

solution instead? This means a) we have the same solution as OG which is nice, and b) I think it's more aligned with what the issue is (that axios needs to be transformed, moduleNameMapper merely mocks axios I think)